test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -250,6 +250,9 @@ spec =
         Test.specify "methods of Integer" <|
             Meta.meta Integer . methods . sort . should_equal ['round', 'truncate']
```

Comment:
Why are the only methods here apparently these 2? IIRC `Integer` has many many more methods. I'm confused.

Reply:
Does it? In such case we need to fix more than just a documentation...

Reply:
Intentional https://github.com/enso-org/enso/blob/e9a92a1fb57d7c4f3acbd1ec304a26a691903128/engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala#L434.

This is because Integer doesn't have builtins like `Integer.+`, we only have `Small_Integer.+`, `Big_Integer.+` or `Decimal.+`. So technically, those methods are being erased during codegen.

Reply:
Reported as 
- #6959